files/rupyainr/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "rupyaINR"
+github = ""
+homepage = "http://www.diagramz.com/"
```

Comment:
missed it, have changed that to correct project website now.
Have committed the changes now